internal/command/init_test.go

```diff
@@ -2921,6 +2921,53 @@ func TestInit_testsWithModule(t *testing.T) {
 	}
 }
 
+// Test variables are handled correctly when interacting with module sources
+func TestInit_moduleSource(t *testing.T) {
```

Comment:
Can we also add a test for TestInit_moduleVersion  now that we support that too please?

Reply:
f3a94486d2